.expeditor/scripts/finish_release/sync_acceptance.toml

```diff
@@ -0,0 +1,39 @@
+format_version = 1
+file_descriptor = "Packages to sync acceptamce from production after release"
+
+[[x86_64-linux]]
```

Comment:
Though it's a bit silly, we should add the Cloud Foundry and Mesos exporter to this list, too; basically, *everything* we make in a release should be represented here.

(Probably worth adding a comment to that effect, too)

Reply:
ok. added packages and comment @christophermaier